host-probes-rid-assets-legacy/app/host-probes-rid-assets.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <RestoreSources>$(MSBuildThisFileDirectory)/../packages</RestoreSources>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="host-probes-rid-assets-mylib-unix" Version="1.0.0" />
+    <PackageReference Include="host-probes-rid-assets-mylib-sdkrid" Version="1.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <RuntimeHostConfigurationOption Include="System.Runtime.Loader.UseRidGraph" Value="true" />
```

Comment:
Is this option the only difference between the two tests?

Reply:
Yes.

Reply:
Could we use only one test and rebuild/run it twice with different configurations to avoid this duplication?

Reply:
I split it in two because the legacy one (which is less important) is known to fail on the .NET 8 stage1 builds.
And the default one (which is important) will pass.

Reply:
Ah, I see. Thank for clarifying that.